examples/diagnostics_and_criticism/model_comparison.ipynb

```diff
@@ -2,26 +2,48 @@
  "cells": [
```

Comment:
<p>There should be no need to ignore warnings in this notebook, ArviZ does not warn about the change in default anymore.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>